src/wp-includes/post-thumbnail-template.php

```diff
@@ -113,6 +113,17 @@ function update_post_thumbnail_cache( $wp_query = null ) {
 	$thumb_ids = array();
 
 	foreach ( $wp_query->posts as $post ) {
+
+		/*
+		 * Check $post is valid post object of WP_Post or not,
+		 * while some cases it can be an array of $posts IDs.
+		 * 
```

Comment:
```suggestion
		 *
```

Fix PHPCS

Reply:
@mukeshpanchal27 fixed and updated test cases.